PromoExchange/batman
# Put sign before dollar sign on shipping options

https://trello.com/c/gPACCDsb
### Testplan
1. Go to company store
2. Enter quantity
3. Select shipping option
4. Ensure lower shipping options (negative) have - sign before dollar
